oshmem/mca/atomic/atomic.h

```diff
@@ -86,17 +86,61 @@ struct mca_atomic_base_module_1_0_0_t {
     opal_object_t super;
 
     /* Collective function pointers */
+    int (*atomic_add)(void *target,
+                      const void *value,
+                      size_t size,
+                      int pe,
+                      struct oshmem_op_t *op);
```

Comment:
can we remove the 'op' arguemnt, now that there is a separate function for every op?

Reply:
yes, but it will affect most of binding, including fortran